nautobot/docs/rest-api/overview.md

```diff
@@ -441,16 +445,16 @@ GET /api/ipam/prefixes/7d2d24ac-4737-4fc1-a850-b30366618f3d/
     "tenant": null,
     "vlan": null,
     "status": {
-        "value": "container",
-        "label": "Container"
+        "value": "reserved",
+        "label": "Reserved"
     },
```

Comment:
Looks like I missed updating the status representation here and below in #3225. Mind fixing these to use the new NestedStatusSerializer representation?